### ISSUE TITLE
Add Additional Installation Instructions to README.md

### ISSUE THREAD

**Issue by nelsonic** [dwyl/smart-home-firmware#10]
The Scenic drivers:
https://github.com/dwyl/smart-home-firmware/blob/0be0578a77381f17a0edd17882c75b89ae4b33c7/mix.exs#L62
Require a few build dependencies to be installed. see:  https://github.com/boydm/scenic_driver_glfw#installing-on-macos

@th0mas do you remember installing these on your Mac?

**Comment by t0mhaines** [comment]
Yeah, currently documented in https://github.com/dwyl/learn-scenic.

I need to update some documentation anyway, and its probably worth adding this to `getting_started.md` in `dwyl/smart_home_security_system`

**Comment by nelsonic** [comment]
On my Mac, when I attempt to run `mix test`, I see the following error:

![image](https://user-images.githubusercontent.com/194400/88399473-d76dd080-cdbe-11ea-8785-3bb3608d891e.png)

**Comment by nelsonic** [comment]
@th0mas right, gotcha: https://github.com/dwyl/learn-scenic#installing
Installing now. ⏳
